central/main.go

```diff
@@ -282,7 +282,7 @@ func servicesToRegister(registry authproviders.Registry, authzTraceSink observe.
 		alertService.Singleton(),
 		apiTokenService.Singleton(),
 		authService.New(),
-		authProviderService.New(registry, groupDataStore.Singleton()),
+		authProviderService.NewWithDefaultProvider(registry, groupDataStore.Singleton()),
```

Comment:
I imagine it'd be cleaner to call into the service (or even datastore) to set up initial auth provider and groups config.